internal/operators/odf/validations.go

```diff
@@ -164,14 +164,14 @@ func (o *operator) setStatusInsufficientResources(odfClusterResources *odfCluste
 }
 
 // count all disks of drive type ssd or hdd
-func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string) (int64, error) {
+func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string, mode odfDeploymentMode) (int64, error) {
 	var countDisks int64
 	var err error
 
 	for _, disk := range disks {
 		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
 			if disk.SizeBytes < conversions.GbToBytes(o.config.ODFMinDiskSizeGB) {
-				err = fmt.Errorf("ODF requires all the non-bootable disks to be more than %d GB", o.config.ODFMinDiskSizeGB)
+				err = fmt.Errorf("Insufficient Resources to deploy ODF in non-bootable %s Mode. ODF requires a minimum of 3 hosts with 1 non-bootable disk on each host of size at least %d GB", mode, o.config.ODFMinDiskSizeGB)
```

Comment:
What does "non-bootable" mean in this context?

Reply:
> What does "non-bootable" mean in this context?

I think the intention when writing "non-bootable" was actually "non-installation disk" AKA disks that are not the installation disk chosen for this host.

"Installation disk" is very different from "Bootable", not just in general, but also in the context of AI - a disk being bootable has a different meaning and consequences for us. 

If we're already changing the message, we should change that as well

Reply:
Also how did "non-bootable disks" (which was wrong to begin with) become "non-bootable <> mode"? What does this even mean?